### PR TITLE
Fix bug causing SDK to make multiple failed connection attempts

### PR DESCRIPTION
**Pull request by jugarrit**
Changing the faye branch to one that is better maintained...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
     "deep-equal": "1.0.1",
-    "faye": "smooch/faye#temp-smooch-fix",
+    "faye": "smooch/faye#faye-fixes",
```
Comment (lemieux) [inline package.json]: maybe it would be easier to target the commit sha instead of creating branches each time..?

Comment (jugarrit) [reply package.json]: My thought was that from now on we'd use this branch. I wanted a place where I could open PRs to if we needed further fixes.

Comment (lemieux) [reply package.json]: Might be easier to trigger new deploys (and see what actually changed) if we change the sha each time though

Comment (jugarrit) [reply package.json]: Sure

**Pull request merged**